conftest.py

```diff
@@ -0,0 +1,114 @@
+                        end_date=timezone.now() + datetime.timedelta(days=3),
+                        price_per_day=100,
+                        )
+
+
+@pytest.fixture
+def dogowner_message_to_daycare1():
+    return Message.create(author=AuthorOptions.DogOwner,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=1),
+                          text='Hello this is the test message1 from owner to day care')
+
+
+@pytest.fixture
+def daycare1_reply_to_dogonwer_message():
+    return Message.create(author=AuthorOptions.DayCare,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=1),
+                          text='This is reply to first message from daycare to owner')
+
+
+@pytest.fixture
+def daycare2_message_to_dogowner():
+    return Message.create(author=AuthorOptions.DayCare,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=2),
+                          text='Hello this new chat between daycare2 and dogowner')
+
+
+@pytest.fixture
+def daycare3_message_to_dogowner():
+    return Message.create(author=AuthorOptions.DayCare,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=3),
+                          text='new chat between daycare3 and dogowner')
+
+
+@pytest.fixture
+def create_dog_owner_user():
+    return DogOwner.create(email='testuser@gmail.com',
+                           username='testuser01',
+                           password='testpassowrd',
+                           dog_name='kliford',
+                           first_name='NEW',
+                           last_name='USER',
+                           phone_number=1234567890,
+                           dog_race='lavrador',
+                           dog_picture_url="https://www.akc.org/wp-content/uploads/2019/06/Bohemian-Shepherd.1.jpg",
+                           dog_age=10,
+                           dog_weight=6,
+                           dog_gender='M'
+                           )
+
+
+@pytest.fixture
+def create_daycare_user():
+    return DayCare.create(email="test@gmail.com", username="testuser01", password="pass",
+                          name="Puppies", description="This is the first daycare test",
+                          price_per_day=10,
+                          capacity=50, area="Merkaz", city="Tel-Aviv", address="The best street 5")
+
+
+@pytest.fixture
+def create_image1(create_daycare_user):
+    return Image.create(url="../../static/images/daycare_image_test_01.jpeg", daycare_id=create_daycare_user)
+
+
+@pytest.fixture
+def create_image2(create_daycare_user):
+    return Image.create(url="../../static/images/daycare_image_test_02.jpeg", daycare_id=create_daycare_user)
+
+
+@pytest.fixture
+def daycare():
```

Comment:
we have the same creation multiple time, please combine
line 72 `def create_daycare_user():`

Reply:
Nice catch! will be changed